jsettlers.common/src/jsettlers/common/menu/action/EActionType.java

```diff
@@ -116,6 +116,11 @@
 	SHOW_SELECTION,
 
 	/**
+	 * Sets the screen to display current message.
+	 */
+	GOTO_MSG,
```

Comment:
I would like to keep this longer and in sync with SHOW_SELECTION. So what about `SHOW_MESSAGE`

We also have `PAN_TO`, so we might fix those names to make them similar.